application/src/mmul.cu

```diff
@@ -1,4 +1,4 @@
-#include "mmul.cuh"
+#include "../include/mmul.cuh"
```

Comment:
Why? `target_include_directory` will sort this out

Reply:
Yea, I guess the IDE did it automatically when I restructured everything.